.prettierignore

```diff
@@ -3,3 +3,6 @@
 /dist/*
 /coverage/*
 /.nyc_output/*
+/.github/*
+LICENSE.md
+README.md
```

Comment:
Why ignore README LICENSE ? Prettier should format this files

Reply:
well, that's true, will change.

Reply:
I think that we can also run prettier vs .yml files, but we will need to add this plugin:
https://github.com/ikatyang/yaml-unist-parser

So we can also get rid of _/.github/*_ from _.prettierignore_